lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,12 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  binary_array.reverse!
```

Comment:
This works, but we'd asked you to it without the enumerable methods.  Instead I suggest making the start index start at the last index and count down to 0.